pkg/driverbuilder/builder/builders.go

```diff
@@ -185,31 +185,31 @@ func (b *Build) setGCCVersion(builder Builder, kr kernelrelease.KernelRelease) {
 	image, ok := b.Images.findImage(b.TargetType, targetGCC)
 	if ok {
 		b.GCCVersion = image.GCCVersion.String()
-	}
-
-	// Step 2:
-	// Build the list of "proposed" GCC versions,
-	// that is, the list of available gccs from images
-	// for each builder image
-	proposedGCCs := make([]semver.Version, 0)
-	for _, img := range b.Images {
-		proposedGCCs = append(proposedGCCs, img.GCCVersion)
-		logger.WithField("image", img.Name).
-			WithField("targetGCC", targetGCC.String()).
-			Debug("proposedGCC=", img.GCCVersion.String())
-	}
+	} else {
```

Comment:
If we found a `gccversion` provided by an image, there is no need to fallback at secondary algorithm.